src/main/java/appeng/tile/storage/TileChest.java

```diff
@@ -224,23 +225,23 @@ private IMEInventoryHandler getHandler( final IStorageChannel channel ) throws C
 			if( !is.isEmpty() )
 			{
 				this.isCached = true;
-				this.cellHandler = AEApi.instance().registries().cell().getHandler( is );
-				if( this.cellHandler != null )
+				ICellHandler cellHandler = AEApi.instance().registries().cell().getHandler( is );
+				if( cellHandler != null )
 				{
 					double power = 1.0;
 
-					final IMEInventoryHandler<IAEItemStack> itemCell = this.cellHandler.getCellInventory( is, this,
+					final ICellInventoryHandler<IAEItemStack> itemCell = cellHandler.getCellInventory( is, this,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "itemCell" which hides the field declared at line 125. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)